### PR TITLE
added os check in pf9ctl_setup

### PR DESCRIPTION
**Pull request by devidask27**
Screenshot 2021-02-26 at 4 09 41 PM

### CODE REVIEW

**Comment by devidask27** [comment]
TC build link
http://teamcity.platform9.horse:8111/buildConfiguration/Pf9project_Platform9Components_Pf9ctl/1449504

File: pf9ctl_setup [hunk 1]
```diff
@@ -95,6 +95,15 @@ cat << "EOF"
 EOF
 }
 
+checkOS_version() {
+    cat /etc/*release &>/dev/null
+    if [ $? -eq 1 ];
```
Comment (anupbarve) [inline pf9ctl_setup:100]: Should check for not equal to 0 instead.

File: pf9ctl_setup [hunk 2]
```diff
@@ -95,6 +95,15 @@ cat << "EOF"
...
+    then
+        echo "This OS is not supported"
```
Comment (anupbarve) [inline pf9ctl_setup:102]: Add a . at the end.

**Review by anupbarve — CHANGES_REQUESTED** [review]

File: pf9ctl_setup (hunk 2 above)
Comment (devidask27) [reply pf9ctl_setup]: Done

File: pf9ctl_setup (hunk 1 above)
Comment (devidask27) [reply pf9ctl_setup]: Done

**Review by anupbarve — APPROVED** [review]

**Pull request merged**